charts/chronicle/templates/_chronicle.tpl

```diff
@@ -68,8 +68,12 @@ http://{{ include "chronicle.id-provider.service" . }}:8090/jwks
 {{- end -}}
 
 {{- define "chronicle.userinfoUrl" -}}
+{{- if .Values.devIdProvider.enabled -}}
```

Comment:
Not good enough - you need the similar logic as for `chronicle.jwksUrl`  It's legitimate to keep the devIdProvider around, but use a different URL.

Reply:
The changes didn't "outdate" your comment, but I think it's been addressed.